File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
    string answer = "";
    
    if(isalpha(s[0])){
        answer += toupper(s[0]);
    } else{
        answer += s[0];
    }   
    
    for(int i=1; i<s.size(); i++){
        if(s[i-1] == ' ' && isalpha(s[i])){
            answer += toupper(s[i]);
            continue;
        } 
        answer += tolower(s[i]);
         
    }
    cout << answer;
    
    return answer;
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = s.split(" ");
    
    for(let i=0; i<answer.length; i++){
        
        answer[i] = answer[i].substring(0,1).toUpperCase() + answer[i].substring(1).toLowerCase();
        
    }
    
    return answer.join(" ");
}